### PR TITLE
Fill in the Metrics details in the Getting Started guide.

### PR DESCRIPTION
**Pull request by malafeev**
Updates #1002 
Some copy paste from Python's guide https://github.com/open-telemetry/opentelemetry-python/blob/master/docs/getting-started.rst#adding-metrics
with Java example

### CODE REVIEW

File: QUICKSTART.md [hunk 1]
```diff
@@ -225,6 +225,36 @@ public void handle(HttpExchange he) {
...
+        .setDescription("Processed jobs")
+        .setUnit("1")
+        .setLabelKeys(Collections.singletonList("Key"))
+        .build();
+
+// It is recommended that the API user keep a reference to a Bound Counter
+BoundLongCounter someWorkBound = counter.bind("SomeWork");
+
+// Record data
+someWorkBound.add(123);
```
Comment (arminru) [inline QUICKSTART.md]:
```suggestion
BoundLongCounter someWorkCounter = counter.bind("Key", "SomeWork");

// Record data
someWorkCounter .add(123);
```

`bind` needs to be supplied with a list of key/value pairs, not just the values.
Also the use of "Bound" as a subject name was dropped a while ago.

File: QUICKSTART.md [hunk 2]
```diff
@@ -225,6 +225,36 @@ public void handle(HttpExchange he) {
...
+business metric such as transactions.
+
+All metrics can be annotated with labels: additional qualifiers that help describe what
+subdivision of the measurements the metric represents.
+
+The following is an example of metric usage:
+
+```java
+// Gets or creates a named meter instance
+Meter meter = OpenTelemetry.getMeterProvider().get("library_name");
```
[the remark this answers is not in the document]
Comment (malafeev) [reply QUICKSTART.md]: done

File: QUICKSTART.md (hunk 1 above)
Comment (malafeev) [reply QUICKSTART.md]: updated

Comment (thisthat) [reply QUICKSTART.md]: The proper code woudl be:  
```java
BoundLongCounter someWorkBound = counter.bind(meter.createLabelSet("Key", "SomeWork"));
```

Comment (malafeev) [reply QUICKSTART.md]: method `meter.createLabelSet(...)` doesn't exist

Comment (arminru) [reply QUICKSTART.md]: @thisthat not anymore since LabelSet was removed in #992

Comment (thisthat) [reply QUICKSTART.md]: Ops! I have to get up2date with the Metric API then! :sweat_smile:

File: QUICKSTART.md [hunk 3]
```diff
@@ -225,6 +225,36 @@ public void handle(HttpExchange he) {
...
+```java
+// Gets or creates a named meter instance
+Meter meter = OpenTelemetry.getMeterProvider().get("instrumentation-library-name","semver:1.0.0");
+
+// Build counter e.g. LongCounter 
+LongCounter counter = meter
+        .longCounterBuilder("processed_jobs")
+        .setDescription("Processed jobs")
+        .setUnit("1")
+        .setLabelKeys(Collections.singletonList("Key"))
```
Comment (bogdandrutu) [inline QUICKSTART.md]: Consider to remove for the moment the set label key. For the moment we don't do anything with that.

Comment (malafeev) [reply QUICKSTART.md]: done

File: QUICKSTART.md [hunk 4]
```diff
@@ -225,6 +225,35 @@ public void handle(HttpExchange he) {
...
+
+// Build counter e.g. LongCounter 
+LongCounter counter = meter
+        .longCounterBuilder("processed_jobs")
+        .setDescription("Processed jobs")
+        .setUnit("1")
+        .build();
+
+// It is recommended that the API user keep a reference to a Bound Counter
+BoundLongCounter someWorkCounter = counter.bind("Key", "SomeWork");
```
Comment (bogdandrutu) [inline QUICKSTART.md]: We should specify that we expect this to stay alive for the entire time or call unbind when no-longer needed.

Comment (malafeev) [reply QUICKSTART.md]: updated

**Comment by jkwatson** [comment]
Would love to see at least one example of non-bound usage, and an observer.

**Comment by bogdandrutu** [comment]
@jkwatson updated the description to not mark the issue as fixed, so we can add more examples :)

File: QUICKSTART.md [hunk 5]
```diff
@@ -225,6 +225,36 @@ public void handle(HttpExchange he) {
...
+business metric such as transactions.
+
+All metrics can be annotated with labels: additional qualifiers that help describe what
+subdivision of the measurements the metric represents.
+
+The following is an example of metric usage:
+
+```java
+// Gets or creates a named meter instance
+Meter meter = OpenTelemetry.getMeterProvider().get("instrumentation-library-name","semver:1.0.0");
```
Comment (jkwatson) [inline QUICKSTART.md]: I'd remove the "semver:" from the beginning of the version.

Comment (bogdandrutu) [reply QUICKSTART.md]: That is in the version semantic convention

**Pull request merged**